.github/workflows/build-common.yml

```diff
@@ -42,6 +42,12 @@ jobs:
           distribution: temurin
           java-version: 17
 
+      - name: Using a local standalone exporter dependency?
+        if: ${{ hashFiles('azure-monitor-opentelemetry-exporter-1.0.0-beta.*.jar') != '' }}
+        run: |
+          mvn -version
+          mvn install:install-file -Dfile="azure-monitor-opentelemetry-exporter-1.0.0-beta.22.jar" -DpomFile="azure-monitor-opentelemetry-exporter-1.0.0-beta.22.pom" -DgroupId="com.azure" -DartifactId="azure-monitor-opentelemetry-exporter" -Dversion="1.0.0-beta.22"
```

Comment:
need to dynamically figure out the version. i can read the gradle.property inside the jar.. but do you think there is an easier way?